lib/manager/buildkite/__fixtures__/pipeline3.yml

```diff
@@ -8,14 +8,6 @@ steps:
 
   - wait
 
-  # Use the app image built above to run concurrent tests
-  - name: "Docker Test %n"
-    command: test.sh
-    parallelism: 25
-    plugins:
-      https://github.com/buildkite/plugin-docker-compose#v1.3.2:
-        run: app
-
```

Comment:
Why was this removed?

Reply:
The test that uses that sample pipeline is testing whether the correct "skip reasons" are added. With the functionality I introduced, this plugin should no longer be skipped, so I figured it would make sense to remove it from the test.